dependency-check-maven-config.xml

```diff
@@ -1,5 +1,94 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://www.owasp.org/index.php/OWASP_Dependency_Check_Suppression">
+
+    <!-- tika -->
+    <suppress>
+        <notes>
+            This is an issue with the jmatio library included in the tika-parser 1.13. The problem
+            doesn't exist in 1.14/1.15 so this is a false positive.
+            tika-bundle-1.15.0_1.jar: vorbis-java-tika-0.8.jar: CVE-2016-6809
+        </notes>
+        <cve>CVE-2016-6809</cve>
+    </suppress>
+    <suppress>
+        <notes>
+            False positive. The cve is for jcore which is not used by tika-bundle or apache-mime4j-core
+            tika-bundle-1.14.0_1.jar: apache-mime4j-core-0.7.2.jar: CVE-2012-4232
+        </notes>
+        <cve>CVE-2012-4232</cve>
+    </suppress>
+    <!-- Remove this suppression CVE-2017-5644 once DDF-3064 has been resolved-->
```

Comment:
❓ Did we mention that Alliance also needs to be updated in that DDF ticket? Or should we have a separate Alliance ticket?